source/fundamentals/serialization/poco.txt

```diff
@@ -247,7 +248,7 @@ The following code sample maps the ``Identifier`` property to the
        public string Identifier { get; set; }
    }
 
-.. warning:: Multiple Id Fields
+.. warning:: Multiple ID Fields
```

Comment:
Changed most occurrences of 'Id' to 'ID' since you can use any name for an ID property

Reply:
But... the C#/.Net naming convention is to use `Id`.

Reply:
After talking to @jordan-smith721, keeping as 'ID' to comply with our Style Guide